site/static/main.js

```diff
@@ -1,36 +1,34 @@
 /* tslint:disable:no-unused-variable */
```

Comment:
Oh man.... that's another point in case of an overkill..

That file can easily be a JS and we can remove one more ugly npm script.

"build:test-gallery": "browserify test-gallery/main.ts -p [tsify -p test-gallery] -d > test-gallery/main.js",